compiler/modules/front.syntax/ast.bal

```diff
@@ -46,7 +46,7 @@ public type ConstDefn record {|
 |};
 
 public type Stmt VarDeclStmt|AssignStmt|CallStmt|ReturnStmt|IfElseStmt|MatchStmt|WhileStmt|ForeachStmt|BreakStmt|ContinueStmt|CompoundAssignStmt|PanicStmt;
-public type CallStmt FunctionCallExpr|MethodCallExpr;
+public type CallStmt FunctionCallExpr|MethodCallExpr|CheckingExpr;
```

Comment:
There should be a subtype of CheckingExpr that corresponds to the kind of CheckingExpr allowed in CallStmt